src/components/Search/FrameworkDropdown.tsx

```diff
@@ -74,8 +67,7 @@ function FrameworkDropdown({
 
   const styles = {
     container: style({
-      minWidth: `${compact ? '280' : '319'}px !important`,
-
+      minWidth: '319px !important',
```

Comment:
I'm not a fan of `!important` as it usually indicates a bigger problem with the design of cascading styles, but it's fine here I guess since we already had it :)